Automattic/wp-calypso
# Add card for Jetpack legacy products

### Changes proposed in this Pull Request

This PR adds a prop to JetpackPlanCard to show it as deprecated.

### Testing instructions

- Download the PR
- Visit `/devdocs/design/jetpack-plan-card`
- Check that the deprecated variant is shown and that the coloured elements (except the icon) are yellow (`--color-yellow-40` or `#c08c00`)

### Screenshots
<img width="689" alt="Screen Shot 2020-08-05 at 4 38 10 PM" src="https://user-images.githubusercontent.com/1620183/89461515-16464380-d73a-11ea-809b-dbef56de9593.png">
